src/main/java/net/liquidpineapple/pang/screens/LevelEditor.java

```diff
@@ -0,0 +1,227 @@
+      ex.printStackTrace();
+    }
+  }
+
+  /**
+   * Method that updates the Leveleditor.
+   */
+  @Override
+  public void doUpdate() {
+    super.doUpdate();
+    PointerInfo pointerInfo = MouseInfo.getPointerInfo();
+    Point point1 = pointerInfo.getLocation();
+    Point point2 = Application.getBoard().getLocationOnScreen();
+    int intXpos = (int) (point1.getX() - point2.getX());
+    int intYpos = (int) (point1.getY() - point2.getY());
+    if (InputHandler.isAnyKeyPressed()) {
+      BallMovement ballMovement = BallMovement.valueOf("LEFT_MOVEMENT");
+      //key 1 pressed, add ball with size 1
+      if (InputHandler.getKeysDown().contains(49)) {
+        InputHandler.getKeysDown().clear();
+        int size = 1;
+        int offset = 0;
+        try {
+          Image ballImage = ImageIO.read(LevelScreen.class.getResource("/sprites/Balls/green.png"));
+          offset = ballImage.getWidth(null) / 2;
+        } catch (IOException ioException) {
+          ioException.printStackTrace();
+        }
+        addedObjects.add(new Ball(intXpos - offset, intYpos - offset, ballMovement, size));
+      }
+      //key 2 pressed, add ball with size 2
+      if (InputHandler.getKeysDown().contains(50)) {
+        InputHandler.getKeysDown().clear();
+        int size = 2;
+        int offset = 0;
+        try {
+          Image ballImage = ImageIO.read(LevelScreen.class.getResource("/sprites/Balls/red.png"));
+          offset = ballImage.getWidth(null) / 2;
+        } catch (IOException ioException) {
+          ioException.printStackTrace();
+        }
+        addedObjects.add(new Ball(intXpos - offset, intYpos - offset, ballMovement, size));
+      }
+      //key 3 pressed, add ball with size 3
+      if (InputHandler.getKeysDown().contains(51)) {
+        InputHandler.getKeysDown().clear();
+        int size = 3;
+        int offset = 0;
+        try {
+          Image ballImage = ImageIO.read(LevelScreen.class
+              .getResource("/sprites/Balls/yellow.png"));
+          offset = ballImage.getWidth(null) / 2;
+        } catch (IOException ioException) {
+          ioException.printStackTrace();
+        }
+        addedObjects.add(new Ball(intXpos - offset, intYpos - offset, ballMovement, size));
+      }
+      //key 4 pressed, add ball with size 4
+      if (InputHandler.getKeysDown().contains(52)) {
+        InputHandler.getKeysDown().clear();
+        int size = 4;
+        int offset = 0;
+        try {
+          Image ballImage = ImageIO.read(LevelScreen.class.getResource("/sprites/Balls/blue.png"));
+          offset = ballImage.getWidth(null) / 2;
+        } catch (IOException ioException) {
+          ioException.printStackTrace();
+        }
+        addedObjects.add(new Ball(intXpos - offset, intYpos - offset, ballMovement, size));
+      }
+      //key 5 pressed, add player
+      if (InputHandler.getKeysDown().contains(53) && !addedPlayer) {
+        InputHandler.getKeysDown().clear();
+        addedPlayer = true;
```

Comment:
2 player support will not be available in this current build. May be added in another sprint?